sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1115,3 +1115,333 @@ def max_deflection(self):
             return (points[deflections.index(max_def)], max_def)
         else:
             return None
+
+
+
+class Beam_3d(Beam):
```

Comment:
Don't use underscores in class names. This should be "Beam3D".

Reply:
Oh, I don't know what I was thinking while using this name. Would change this in any of the existing PR.